streams-tests/jvm/src/test/scala/zio/stream/experimental/ZSinkPlatformSpecificSpec.scala

```diff
@@ -0,0 +1,25 @@
+package zio.stream.experimental
+
+import zio._
+import zio.test._
+import zio.test.Assertion._
+import java.io.ByteArrayOutputStream
+
+object ZSinkPlatformSpecificSpec extends ZIOBaseSpec {
+
+  override def spec: ZSpec[Any, Throwable] = suite("ZSink JVM experimental")(
+    suite("fromOutputStream")(
+      test("writes to byte array output stream") {
+        val data = (0 to 100).mkString
+
+        for {
+          bytes  <- Task(data.getBytes("UTF-8"))
+          os      = new ByteArrayOutputStream(data.length)
+          length <- ZStream.fromIterable(bytes).run(ZSink.fromOutputStream(os))
+          str    <- Task(os.toString("UTF-8"))
```

Comment:
I would use the constant defined in java.nio.charset.StandardCharset

Reply:
I tried it, but I got a compilation error on 19 line:
```
ZSinkPlatformSpecificSpec.scala:20:29: overloaded method toString with alternatives:
[error]   (x$1: Int)String <and>
[error]   (x$1: String)String
[error]  cannot be applied to (java.nio.charset.Charset)
[error]           str    <- Task(os.toString(UTF_8))
[error]                             ^
[error] one error found
```
It's strange because in source there is overloaded method toString parameterized by java.nio.charset.Charset.
That's why I decided to use String literal instead.